### ISSUE TITLE
result is: "Commit message could not be generated."

### ISSUE THREAD

**Issue by pauledenburg** [Blarc/ai-commits-intellij-plugin#51]
When generating a commit message, the result is: "Commit message could not be generated."

There is no other information displayed.

<img width="1477" alt="image" src="https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/f8e49dab-d0e0-442d-9048-2eea1d5a7e48">

**Comment by pauledenburg** [comment]
When testing with [aicommits](https://github.com/Nutlope/aicommits) I noticed that I was over my quota (error 429).

I added payments and switched to paid account over at OpenAI.

aicommits is now able to make requests to OpenAI and generate commit messages.

PHPStorm with AICommits still generate the 'Commit message could not be generated'

**Comment by Blarc** [comment]
@pauledenburg Sorry for responding so late. Have you maybe managed to resolve the problem with PHPStorm?

Regarding the "no other information displayed", there was no notification? I call the generate commit message in a try catch block:
```kotlin
            runBlocking(Dispatchers.Main) {
                try {
                    val generatedCommitMessage = openAIService.generateCommitMessage(prompt, 1)
                    commitMessage.setCommitMessage(generatedCommitMessage)
                    AppSettings.instance.recordHit()
                } catch (e: Exception) {
                    commitMessage.setCommitMessage(message("action.error"))
                    sendNotification(Notification.unsuccessfulRequest(e.localizedMessage ?: message("action.unknown-error")))
                }
            }
```
And would expect that a notification would pop up.

**Comment by pauledenburg** [comment]
No problem @Blarc 

The issue still exists. I have attached a screen recording where I hit Cmd+K to commit changed files and then click the button to have a commit-message generated.

I do not see any messages popup.
I have version 1.2.0 of the plugin installed.

https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/392cb1a5-f4b3-4a43-936d-84a809df49cd

**Comment by Blarc** [comment]
@pauledenburg Which version of **PhpStorm** are you using? Could you try to verify the token by clicking the button `Verify` in settings? 

I will release new version **1.2.1** that will write the error message as the commit message when exception occurs. Hopefully this will make it clear what's going on.

**Comment by pauledenburg** [comment]
@Blarc  I'm using PHPStorm version 2023.1.2 Build #PS-231.9011.38, built on May 16, 2023

I installed your new version via the zip-file and now I do get additional information:

`This is not a chat model and thus not supported in the v1/chat/completions endpoint. Did you mean to use v1/completions?`

<img width="1393" alt="image" src="https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/d82b3616-d2bc-45d4-871c-ae64d37d2062">

I tried changing the configuration for the plugin by setting the `OpenAI Host` to `https://api.openai.com/v1/completions` as per the error message, but that did not change the error.

So I guess I should not use `Ada` as model? It was the default selected during installation.

<img width="1292" alt="image" src="https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/129d6d6e-7c76-4ee3-8345-13aad37aecc2">

**Comment by Blarc** [comment]
@pauledenburg Can you try these settings?
![image](https://github.com/Blarc/ai-commits-intellij-plugin/assets/36704759/4ada2e3a-2615-4cce-b59f-d150c284fe48)

Default model should be `gpt-3.5-turbo`. I have no idea why it was `ada` for you. Also mind the `OpenAI host`, my url does not contain `v1`.

**Comment by pauledenburg** [comment]
@Blarc I updated the settings conform your printscreen. 

But now it won't validate when I click the 'Verify' button.

<img width="982" alt="image" src="https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/4b0439a5-3180-4ffe-8f7f-ce019d71df44">

This is what I get when I want to commit.

<img width="1202" alt="image" src="https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/2c816775-fe59-435e-82bc-468039c061bd">

**Comment by Blarc** [comment]
@pauledenburg I'm sorry, the url `https://api.openai.com/v1/` is correct. It seems it has changed over time and my settings weren't updated. Does it maybe work with this url and `gpt-3.5-turbo` model?

**Comment by pauledenburg** [comment]
@Blarc that worked! Note that the last forward slash `/` in the url is important. Without it, I got the same error `OpenAI configuration is not valid (404)`.

Requesting a commit message then delivered the error that the input length was exceeded.

![image](https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/a569609f-67ee-417c-b394-1d840a00dcda)

By just selecting a few changed files, I did get a working message.

![image](https://github.com/Blarc/ai-commits-intellij-plugin/assets/2551131/cfd07ff5-0165-47d5-8ec0-851fc28ce66a)

Thanks for helping me out!
